src/schemas/replicated.json

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
Where did this file come from? Was this committed by accident?

Reply:
I didn't touch that file. How can I get rid of it? Should I just delete that?